README.md

```diff
@@ -14,19 +14,26 @@
 
 ## Live Demo 🔗
 
-[Live Demo Link](https://codepantherr.github.io/Portfolio/)
+[Live Demo Link](https://codepantherr.github.io/leaderboard/)
```

Comment:
[OPTIONAL] If you still haven't deployed I think it would be better to just delete this section of the README file until later.

Reply:
Okay